aidants_connect_web/templates/aidants_connect_web/new_mandat/new_mandat.html

```diff
@@ -58,18 +61,16 @@ <h2>Étape 1 : Sélectionnez la ou les démarche(s)</h2>
                   <label class="label-demarche" for="button-{{ value }}">
                     <img src="{{ label.icon }}" alt=""/>
                     <strong>{{ label.titre }}</strong>
-                    <p>
-                      {{ label.description }}
-                      {% if label.service_exemples %}
-                        <br/>
-                        <small title="exemples de services publics">
-                          <i>Exemples :</i>
-                          {% for service in label.service_exemples %}
-                            <span class="label label-small">{{ service }}</span>
-                          {% endfor %}
-                        </small>
-                      {% endif %}
-                    </p>
+                    {{ label.description }}
+                    {% if label.service_exemples %}
+                      <br/>
+                      <small title="exemples de services publics">
```

Comment:
On peut se passer de ce `title`, tiens.
```suggestion
                      <small>
```